usr/sbin/revpi-bthelper

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# referenced pi-bluetooth, bluetooth helper for Revolution Pi BT
+. /usr/bin/revpi-common
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+	echo "Usage: $0 <bluetooth hci device>"
+	exit 0
+fi
+
+dev=$1
+
+# Need to bring hci up before looking at MAC as it can be all zeros during init
+# TODO: address prefix for Revpi devices needed.
+if ! /bin/hciconfig "$dev" | grep -q "BD Address: B8:27:EB"; then
```

Comment:
I'm bnot sure if we need to check the BDADDR. All devices should have a proper address from murata.
Also the address you are matching is from the Raspberry Pi Foundation.

Reply:
I think you wanted to copy this line
```
if ! /bin/hciconfig $dev | grep -q "Bus: UART"; then
```

Reply:
ok